pkgs/by-name/me/meson/setup-hook.sh

```diff
@@ -58,6 +58,14 @@ mesonCheckPhase() {
         flagsArray+=("--timeout-multiplier=0")
     fi
 
+    # Parallel building is enabled by default.
+    local buildCores=1
+    if [ "${enableParallelBuilding-1}" ]; then
```

Comment:
Do this work?

If yes, can you make the check more explicit for readability?

Reply:
What do you mean? This is the same way the check is written in every other setup hook, so I don't think changing that is really in scope here.

Reply:
#TIL  

ripgrepping through nixpkgs I found a few examples. This is not the only way out there but it's already used so fine by me (ACK?).

there is also:
- `if [[ -n "${enableParallelBuilding-}" ]]; then`
- `if [[ -z ${enableParallelBuilding-} ]]; then`
- `''${enableParallelBuilding:+-j$NIX_BUILD_CORES}"` which expands into a flag
- `if [[ ! -v enableParallelBuilding || -n "${enableParallelBuilding-}" ]]; then`: which I found in haredo